deploy/modules/aliyun/tidb-operator/manifest/global-job-controller.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: apps/v1
```

Comment:
I'm hesitant to add this job. Job runs asynchronously, we cannot guarantee the kubelets are properly configured when the terraform operation completes. And introducing an uncontrolled process add instabilities to the script.

Possible alternatives:
- update the `attach_node.sh` just like https://github.com/pingcap/tidb-operator/blob/36f505cd2ef32e4d8f25570f70bdb97dafe7f999/deploy/modules/aliyun/tidb-cluster/templates/user_data.sh.tpl#L11 do, which I don't recommend;
- maintain `attach_node.sh` on ourselves.

How do you think?